### PR TITLE
bugfix/bundle-constants added support for const values with bundle na…

### PR DESCRIPTION
**Pull request by oliwier-spryker**
Added support for constants with bundle prefixes, eg const FILESYSTEM_SERVICE = 'FILESYSTEM:FILESYSTEM_SERVICE';

### CODE REVIEW

**Comment by ThomasKuehnel** [comment]
Isn't that a duplication of https://github.com/spryker/architecture-sniffer/pull/16

File: src/Shared/BundleConstantsRule.php [hunk 1]
```diff
@@ -57,4 +59,64 @@ public function apply(AbstractNode $node)
...
+    /**
+     * @param string $value
+     *
+     * @return string
+     */
+    protected function getBundleConstant($value)
+    {
+        $pos = strpos($value, ':');
+        if ($pos === false) {
+            return false;
```
Comment (darius-telycenas) [inline src/Shared/BundleConstantsRule.php]: Either this line is wrong, or `DocBlock` is incomplete…

Comment (dereuromark) [inline src/Shared/BundleConstantsRule.php]: Should probably be `null`. And docblock adding |null .

Comment (dereuromark) [inline src/Shared/BundleConstantsRule.php]: Doesnt even look like this method is used anywhere. @oliwier-spryker

**Comment by stereomon** [comment]
Duplication of https://github.com/spryker/architecture-sniffer/pull/16

**Pull request closed without merging**